docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,288 @@
+        },
+        {
+          "@id": "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMw==:anNvbi0xLXBhcGVy:NDQ1ZWRlZDMtNjNlMy00ZDg0LTkxMmUtYWFjZWM0MjdkMDQ4",
+          "@type": "odrl:Set",
+          "odrl:permission": {
+            "odrl:action": {
+              "odrl:type": "http://www.w3.org/ns/odrl/2/use"
+            },
+            "odrl:constraint": {
+              "odrl:and": {
+                "odrl:leftOperand": "https://w3id.org/catenax/policy/v1.0.0/FrameworkAgreement",
+                "odrl:operator": {
+                  "@id": "odrl:eq"
+                },
+                "odrl:rightOperand": "traceability:v1.0"
+              }
+            }
+          },
+          "odrl:prohibition": [],
+          "odrl:obligation": []
+        },
+        {
+          "@id": "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMQ==:anNvbi0xLXBhcGVy:ODFkMDI2MWYtNDNlNi00ZTIxLWJkMWYtZmFmZTI3MWQwYzhj",
+          "@type": "odrl:Set",
+          "odrl:permission": {
+            "odrl:action": {
+              "odrl:type": "http://www.w3.org/ns/odrl/2/use"
+            },
+            "odrl:constraint": {
+              "odrl:and": [
+                {
+                  "odrl:leftOperand": "https://w3id.org/catenax/policy/v1.0.0/FrameworkAgreement",
+                  "odrl:operator": {
+                    "@id": "odrl:eq"
+                  },
+                  "odrl:rightOperand": "traceability:v1.0"
+                },
+                {
+                  "odrl:leftOperand": "https://w3id.org/catenax/policy/v1.0.0/ContractReference",
+                  "odrl:operator": {
+                    "@id": "odrl:eq"
+                  },
+                  "odrl:rightOperand": "contract-123456789"
+                },
+                {
+                  "odrl:leftOperand": "https://w3id.org/catenax/policy/v1.0.0/UsagePurpose",
+                  "odrl:operator": {
+                    "@id": "odrl:eq"
+                  },
+                  "odrl:rightOperand": "ID 3.1 Trace"
+                }
+              ]
+            }
+          },
+          "odrl:prohibition": [],
+          "odrl:obligation": []
+        }
+      ],
+      "dcat:distribution": [
+        {
+          "@type": "dcat:Distribution",
+          "dct:format": {
+            "@id": "HttpData-PULL"
+          },
+          "dcat:accessService": "911f5da0-c9ee-4259-9a95-39428d08f767"
+        }
+      ],
+      "version": 1.0,
+      "content-type": "application/json",
+      "name": "json-1-paper",
+      "description": "Asset json-1-paper for test purposes",
+      "id": "json-1-paper"
+    }
+  ],
+  "dcat:service": {
+    "@id": "911f5da0-c9ee-4259-9a95-39428d08f767",
+    "@type": "dcat:DataService",
+    "dct:terms": "connector",
+    "dct:endpointUrl": "https://provider-dsp-end.point/api/v1/dsp"
+  },
+  "participantId": "PROVIDER-BPNL"
+}
+
+```
+
+These four Offers are of `@type` either `odrl:Offer` or one of its subclasses. More interesting however are the
```

Comment:
what could be a subclass of Offer? Offer is a subclass of Policy.

Reply:
Good catch - this is in fact wrong. 

However, I'm unsure what's right. In the EDC, `odrl:hasPolicy` currently points to `odrl:Set`s (at least the way it's propagated to the Consumer through the Mgmt API). The DSP says, it's supposed to point to `odrl:Offer`. Both are subclasses of `odrl:Policy` but describe separate concepts.